### DISCUSSION TITLE
Two point correlation function for reconstructing image

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/porespy#915] [Q&A]
Hi Prof. I want to evaluate the reconstruction effect of SNOW algorithm using two-point correlation function. However, the example in the porespy document only contains the input 3D image, not the reconstructed image. The method I am using is `SNOW network extraction (advanced)`. The last step of the network extraction is 
```
regions = watershed(image=-dt, markers=peaks, mask=dt > 0)
regions = randomize_colors(regions)
```
 I use this `regions` image to apply the two-point correlation function and only get a horizontal line. I'm guessing this is not a reconstructed image, so how do I get a reconstructed image? Thanks in advance for your reply.

**Comment by jgostick** [comment]
I am not quite sure what you're hoping to learn by doing a 2point correlation on a watershed image, but I don't think it will work.  The image will have values from 0 to N, while the 2point correlation works on boolean images.
